spec/factories/user_factory.rb

```diff
@@ -3,7 +3,11 @@
     email { "no-reply@sciencehistory.org"}
 
     factory :admin_user do
-      admin { true }
+      user_type { "admin" }
+    end
+
+    factory :editor_user do
+      user_type { "editor" }
     end
```

Comment:
If this isn't actually being used ANYWHERE yet, maybe it's not needed? 

I could see actually *changing* the existing `:user` factory to `:editor_user`, and changing all uses of it in all code to be that... since all current `create(:user)` calls are really meant to create an editor user? Maybe in a next PR?

Reply:
No you're right. We'll need it later but including it in this PR was a mistake.